common-util/src/main/java/org/sunbird/common/models/util/ProjectUtil.java

```diff
@@ -981,6 +981,12 @@ public static Map convertJsonStringToMap(String jsonString) throws IOException {
    * @return request object of target type.
    */
   public static <T> T convertToRequestPojo(Request request, Class<T> clazz) {
-    return (T) mapper.convertValue(request.getRequest(), clazz);
+    return mapper.convertValue(request.getRequest(), clazz);
+  }
+
+  public enum OperationType {
```

Comment:
What is this enum for? The name "OperationType" is very generic. Where are the operation values for rest of the APIs?